### PR TITLE
report: coding period report for week 3

### PR DESCRIPTION
**Pull request by dushimsam**
## Project

Gsoc'23 coding period report

### Changes

Added new pictures, and created a new .md file for documenting the report.

### CODE REVIEW

File: docs/2023/rest/updates/samuel/2023-06-15.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+* Added the REST API to remove a particular main license from the upload: [#2463](https://github.com/fossology/fossology/pull/2463)
+* Added the REST API to retrieve all the main licenses assigned on a particular upload: [#2465](https://github.com/fossology/fossology/pull/2465)
+* Added the REST API to add new license decision for an item: [#2466](https://github.com/fossology/fossology/pull/2466)
+* Added the REST API to delete a specific license decision assigned on an item:  [#2469](https://github.com/fossology/fossology/pull/2469)
+* Added the REST API to list the license decisions for a specific item:  [#2470](https://github.com/fossology/fossology/pull/2470)
+* Added the REST API to update the clearing info (**License text**, **acknowledgement** & **Comment**) for a specific license decision on the item:  [#2471](https://github.com/fossology/fossology/pull/2471)
+* Added the REST API to retrieve the clearing history for a specific item: [#2477](https://github.com/fossology/fossology/pull/2477)
+
+### SOME SCREENSHOTS
```
Comment (avinal) [inline docs/2023/rest/updates/samuel/2023-06-15.md:26]:
```suggestion
### Screenshots
```

File: docs/2023/rest/updates/samuel/2023-06-15.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+#### 5. GET ALL LICENSE DECISIONS FOR AN ITEM API `/uploads/{id}/items/{itemId}/licenses`
+
+![get-license-decisions](/img/reactUI/api/License/get_all_license_decisions.png) 
+
+#### 6. UPDATE THE CLEARING INFO API `/uploads/{id}/items/{itemId}/licenses`
+
+![update-license-decision](/img/reactUI/api/License/set_clearing_info.png)
+
+#### 7. GET CLEARING HISTORY API `/uploads/{id}/items/{itemId}/clearing_history`
```
Comment (avinal) [inline docs/2023/rest/updates/samuel/2023-06-15.md:53]: Please keep the header in normal format. No such need for upper-case.

File: docs/2023/rest/updates/samuel/2023-06-15.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- Add the PR for the REST API to schedule the bulk scan.
+- Add the PR for the REST API to handle the three-pointers functionality.
+- Add the REST API to retrieve the positions for the highlighting the content of the file.
+- Add the REST API for the Tree view: Reuse Summary, Scanner Details (GET & for filter)	
+
+
+
+
+
+
```
Comment (avinal) [inline docs/2023/rest/updates/samuel/2023-06-15.md:68]: Properly format your markdown files. You can use the inbuilt VS Code formatter. Or install the Markdown Lint extension.

**Review by avinal — CHANGES_REQUESTED** [review]
Few suggesttions

File: docs/2023/rest/updates/samuel/2023-06-15.md (hunk 2 above)
Comment (avinal) [reply docs/2023/rest/updates/samuel/2023-06-15.md:53]: See https://github.com/fossology/gsoc/pull/155#discussion_r1234784634

Comment (dushimsam) [reply docs/2023/rest/updates/samuel/2023-06-15.md:53]: Great I am correcting it

File: docs/2023/rest/updates/samuel/2023-06-15.md (hunk 3 above)
Comment (dushimsam) [reply docs/2023/rest/updates/samuel/2023-06-15.md]: Great

**Review by GMishx — APPROVED** [review]

**Pull request merged**